python/src/main/api.py

```diff
@@ -35,7 +36,18 @@ def put(self, email, first_name, last_name):
 @api.route('/api/v1/person/<string:email>')
 class People(Resource):
     def delete(self, email):
-        return 200
+        conn = sqlite3.connect("person_and_item.db")
+        cursor = conn.cursor()
+        db = DB(cursor, conn)
+        person = db.get_person(email)
+        if len(person) == 0:
+            logging.error("Couldn't find person with email '%s'" % email)
+            abort(404)
+        else:
+            db.delete_person(email)
```

Comment:
If I delete a person do I want to keep their items in the item table too?

Reply:
good point

Reply:
We definitely need a test for this!